src/NuGet.Core/NuGet.Protocol/PublicAPI/net472/PublicAPI.Shipped.txt

```diff
@@ -2033,15 +2033,6 @@ static readonly NuGet.Protocol.ProxyAuthenticationHandler.MaxAuthRetries -> int
 ~static readonly NuGet.Protocol.ServiceTypes.SearchAutocompleteService -> string[]
 ~static readonly NuGet.Protocol.ServiceTypes.SearchQueryService -> string[]
 ~static readonly NuGet.Protocol.ServiceTypes.SymbolPackagePublish -> string[]
-~static readonly NuGet.Protocol.ServiceTypes.Version200 -> string
-~static readonly NuGet.Protocol.ServiceTypes.Version300 -> string
-~static readonly NuGet.Protocol.ServiceTypes.Version300beta -> string
-~static readonly NuGet.Protocol.ServiceTypes.Version340 -> string
-~static readonly NuGet.Protocol.ServiceTypes.Version470 -> string
-~static readonly NuGet.Protocol.ServiceTypes.Version490 -> string
-~static readonly NuGet.Protocol.ServiceTypes.Version500 -> string
-~static readonly NuGet.Protocol.ServiceTypes.Version510 -> string
-~static readonly NuGet.Protocol.ServiceTypes.Versioned -> string
```

Comment:
While I believe it's very unlikely that anyone using the NuGet.Protocol package in their own app is using any of these APIs, it's a breaking change to move the strings to a different type. In my opinion, the "benefit" doesn't warrant the risk of breaking APIs.